vector/src/main/java/im/vector/app/core/extensions/ViewExtensions.kt

```diff
@@ -58,3 +63,21 @@ fun ImageView.setDrawableOrHide(drawableRes: Drawable?) {
     setImageDrawable(drawableRes)
     isVisible = drawableRes != null
 }
+
+fun View.setAttributeTintedBackground(@DrawableRes drawableRes: Int, @AttrRes tint: Int) {
+    val drawable = ContextCompat.getDrawable(context, drawableRes)!!
+    DrawableCompat.setTint(drawable, context.fetchThemeColor(tint))
+    background = drawable
+}
+
+fun ImageView.setAttributeTintedImageResource(@DrawableRes drawableRes: Int, @AttrRes tint: Int) {
+    val drawable = ContextCompat.getDrawable(context, drawableRes)!!
+    DrawableCompat.setTint(drawable, context.fetchThemeColor(tint))
+    setImageDrawable(drawable)
+}
+
+fun View.setAttributeBackground(@AttrRes attributeId: Int) {
+    val typedValue = TypedValue()
+    context.theme.resolveAttribute(attributeId, typedValue, true)
+    setBackgroundResource(typedValue.resourceId)
+}
```

Comment:
Have you seen the object ThemeUtils? It's not perfect, but there is some similarity with what is done here, maybe we should harmonize the code later.

Reply:
ahhh, I'll update the extensions to call into the ThemeUtils